tools/deploy_ipk.sh

```diff
@@ -1,23 +1,31 @@
 #!/bin/sh -e
 
-SSH_OPTIONS="-oBatchMode=yes"
+usage() {
+    echo "usage: $(basename "$0") [-h]"
+    echo "  options:"
+    echo "      -h|--help - show this help menu"
+    echo "      --certification-mode - enable certification-mode on the target"
+}
 
-if ! ssh "$SSH_OPTIONS" "$1" /bin/true ; then
-    echo "Error: $1 unreachable via ssh"
-    exit 1
-fi
+deploy() {
+    SSH_OPTIONS="-oBatchMode=yes"
```

Comment:
I don't see SSH_OPTIONS used below. BTW what does `BatchMode` do?

Reply:
`BatchMode` doesn't ask for a password (it fails if you need a password).
It's used in the very first check, it's not strictly needed to use it afterwards (although it doesn't hurt).